### DISCUSSION TITLE
Headless not functioning on Nuxt3 (Windows)

### DISCUSSION THREAD

**Discussion by Impala640** [tailwindlabs/headlessui#1952] [Help]
I am receiving the error below whenever I attempt to use headless tailwind on Nuxt 3.0.0 rc 12 on Windows only. The error does not reproduce when on either of my Macs nor when running the code inside a docker container. Unable to determine the root of the issue. Tried reinstalling Windows on the machine I was testing using but such yielded no results. Any help would be appreciated. 
![chrome_cDC0RQcI2i](https://user-images.githubusercontent.com/48142415/197424832-39f51973-a314-4463-b389-4c7a3da2fa47.png)

**Comment by ale87617** [comment]
I had the same issue! After multiple ties, we added '@headlessui/vue' in transpile array keys

export default defineNuxtConfig({
  modules: [
    '@nuxt/content',
    '@nuxtjs/tailwindcss'
  ],
  css: [
    '@/assets/css/main.css',
  ],
  build: {
    transpile: ['@heroicons/vue', **'@headlessui/vue'**]
  }
